scripts/convert-databases.bash

```diff
@@ -0,0 +1,231 @@
+arbitrumdata_status="n/a"
+wasm_status="n/a"
+classicmsg_status="n/a"
+
+checkMissingValue () {
+    if [[ $1 -eq 0 || $2 == -* ]]; then
+        echo "missing $3 argument value"
+        exit 1
+    fi
+}
+
+printStatus() {
+    echo "== Conversion status:"
+    echo "   l2chaindata database: $l2chaindata_status"
+    echo "   l2chaindata database freezer (ancient): $l2chaindata_ancient_status"
+    echo "   arbitrumdata database: $arbitrumdata_status"
+    echo "   wasm database: $wasm_status"
+    echo "   classic-msg database: $classicmsg_status"
+}
+
+printUsage() {
+echo Usage: $0 \[OPTIONS..\]
+    echo
+    echo OPTIONS:
+    echo "--dbconv          dbconv binary path (default: \"$DEFAULT_DBCONV\")"
+    echo "--src             directory containing source databases (default: \"$DEFAULT_SRC\")"
+    echo "--dst             destination directory"
+    echo "--force           remove destination directory if it exists"
+    echo "--skip-existing   skip convertion of databases which directories already exist in the destination directory"
+}
+
+while [[ $# -gt 0 ]]; do
+    case $1 in
+        --dbconv)
+            shift
+            checkMissingValue $# "$1" "--dbconv"
+            dbconv=$1
+            shift
+            ;;
+        --src)
+            shift
+            checkMissingValue $# "$1" "--src"
+            src=$1
+            shift
+            ;;
+        --dst)
+            shift
+            checkMissingValue $# "$1" "--dst"
+            dst=$1
+            shift
+            ;;
+        --force)
+            force=true
+            shift
+            ;;
+        --skip-existing)
+            skip_existing=true
+            shift
+            ;;
+        --help)
+            printUsage
+            exit 0
+            ;;
+        *)
+            printUsage
+            exit 0
+    esac
+done
+
+if $force && $skip_existing; then
+    echo Error: Cannot use both --force and --skipexisting
+    printUsage
+    exit 1
+fi
+
+if ! [ -e "$dbconv" ]; then
+    echo Error: Invalid dbconv binary path: "$dbconv" does not exist
+    exit 1
+fi
+
+if ! [ -n "$dst" ]; then
+    echo Error: Missing destination directory \(\-\-dst\)
+    printUsage
+    exit 1
+fi
+
+if ! [ -d "$src" ]; then
+    echo Error: Invalid source directory: \""$src"\" is missing
+    exit 1
+fi
+
+src=$(realpath $src)
+
+if ! [ -d "$src"/l2chaindata ]; then
+    echo Error: Invalid source directory: \""$src"/l2chaindata\" is missing
+    exit 1
+fi
+
+if ! [ -d $src/l2chaindata/ancient ]; then
+    echo Error: Invalid source directory: \""$src"/l2chaindata/ancient\" is missing
+    exit 1
+fi
+
+if ! [ -d "$src"/arbitrumdata ]; then
+    echo Error: Invalid source directory: missing "$src/arbitrumdata" directory
+    exit 1
+fi
+
+if [ -e "$dst" ] && ! $skip_existing; then
+    if $force; then
+        echo == Warning! Destination already exists, --force is set, this will remove all files under path: "$dst"
+        read -p "are you sure? [y/n]" -n 1 response
```

Comment:
if you have a --force flag, don't ask "are you sure?". force is so other scripts can use it.